docs/articles/discord-guild.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Discord Guild Configuration Guide
+description: LuzFaltex's guide to configuring Discord Guilds
+author: Foxtrek64
+ms.author: Foxtrek_64 # Microsoft employees only
+ms.date: 04/19/2019
+---
+
+# Discord Guild Configuration Guide
+
+This guide is provided by LuzFaltex to outline the way we configure our Discord Guilds (servers) and the reasons behind them. This guide assumes you will be reading this because you have an interest in building a server to our standards, or because you are a LuzFaltex employee tasked with building a new community.
+
+## Initial Setup
+
+We will start out assuming you have already created your guild, chosen its name, and are ready to begin setup. Many of the practices and the reasons behind them are adapted from Microsoft Best Practice for Windows Active Directory. We additionally rely on [NIST INCITS 359-2012](https://standards.incits.org/apps/group_public/project/details.php?project_id=1658).
+
+Going into this guide, it is a good idea to keep in mind these guidelines:
+
+* **Less is more**: You should make the least amount changes as possible. Roles should only be given the bare minimum permissions to allow users with that role to do their job (see the [RBAC Section](#RBAC-Role-Based-Access-Control).) Channel and Channel Category overrides should only differ from role permissions where that channel or channel category differs. For instance, staff-only channels.
```

Comment:
"least amount of changes possible"
Just take out "as" and add "of"
Other than that, everything's good :P